src/Api/ProjectsApi.php

```diff
@@ -114,9 +126,25 @@ public function projectsUserGet(?string $max_version = null, ?int $limit = 20, ?
   /**
    * {@inheritdoc}
    */
-  public function projectsUserUserIdGet(string $user_id, ?string $max_version = null, ?int $limit = 20, ?int $offset = 0, ?string $flavor = null, &$responseCode, array &$responseHeaders)
+  public function projectsUserUserIdGet(string $userId, string $maxVersion = null, ?int $limit = 20, ?int $offset = 0, string $flavor = null, &$responseCode, array &$responseHeaders)
   {
-    // TODO: Implement projectsUserUserIdGet() method.
+    if (null == $maxVersion)
```

Comment:
please keep them in snake_case (which sis our project default)

$responseCode, $responseHeaders are the only exceptions because of a bug in the API generation tool.